CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.16.0 (2020-06-28)
```

Comment:
`2021`

Reply:
Doh. That was wrong for the last three releases..... I've fixed them all.